packages/csharp/PortingAssistant/PortingAssistant.Telemetry/LogWatcher.cs

```diff
@@ -0,0 +1,278 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using Amazon.Runtime;
+using Amazon.Runtime.CredentialManagement;
+using Aws4RequestSigner;
+using ElectronCgi.DotNet;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Serilog;
+
+namespace PortingAssistant.Telemetry
+{
+    class LogWatcher
+    {
+        private Connection _connection;
+        private Program.TelemetryConfiguration telemetryConfiguration;
```

Comment:
Most of our code use _ like _telemetryConfiguration for private variable. Maybe we should adapt that here.